2-Authorization-I/1-call-graph/README.md

```diff
@@ -42,111 +59,122 @@ In addition, this sample also demonstrates how to use the [Microsoft Graph JavaS
 
 ## Prerequisites
 
-- [Node.js](https://nodejs.org/en/download/) must be installed to run this sample.
-- A modern web browser. This sample uses **ES6** conventions and will not run on **Internet Explorer**.
-- [Visual Studio Code](https://code.visualstudio.com/download) is recommended for running and editing this sample.
-- [VS Code Azure Tools](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-node-azure-pack) extension is recommended for interacting with Azure through VS Code Interface.
-- An **Azure AD** tenant. For more information see: [How to get an Azure AD tenant](https://azure.microsoft.com/documentation/articles/active-directory-howto-tenant/)
-- A user account in your **Azure AD** tenant.
+* [Node.js](https://nodejs.org/en/download/) must be installed to run this sample.
+* [Visual Studio Code](https://code.visualstudio.com/download) is recommended for running and editing this sample.
+* [VS Code Azure Tools](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-node-azure-pack) extension is recommended for interacting with Azure through VS Code Interface.
+* A modern web browser.
+* An **Azure AD** tenant. For more information, see: [How to get an Azure AD tenant](https://docs.microsoft.com/azure/active-directory/develop/test-setup-environment#get-a-test-tenant)
+* A user account in your **Azure AD** tenant.
+
+>This sample will not work with a **personal Microsoft account**. If you're signed in to the [Azure portal](https://portal.azure.com) with a personal Microsoft account and have not created a user account in your directory before, you will need to create one before proceeding.
 
-## Setup
+## Setup the sample
 
 ### Step 1: Clone or download this repository
 
 From your shell or command line:
 
 ```console
-    git clone https://github.com/Azure-Samples/tutorial.git
+git clone https://github.com/Azure-Samples/ms-identity-javascript-tutorial.git
 ```
 
-or download and extract the repository .zip file.
+or download and extract the repository *.zip* file.
 
 > :warning: To avoid path length limitations on Windows, we recommend cloning into a directory near the root of your drive.
 
 ### Step 2: Install project dependencies
 
 ```console
-    cd ms-identity-javascript-tutorial
-    cd 2-Authorization-I/1-call-graph
+    cd 2-Authorization-I\1-call-graph
```

Comment:
>cd 2-Authorization-I\1-call-graph [](http://example.com/codeflow?start=4&length=33)

codegen didnt print this irrespective of the "SampleSubPath" being present in sample.json.
please open an issue/task on codegen to investigate

Reply:
done!